config/crd/bases/infrastructure.cluster.x-k8s.io_vspheremachines.yaml

```diff
@@ -514,6 +514,11 @@ spec:
                   a linked clone. This field is ignored if LinkedClone is not enabled.
                   Defaults to the source's current snapshot.
                 type: string
+              tags:
+                description: Tags is an optional set of tags to add to an instance.
```

Comment:
should be "Tags is an optional set of tag IDs to add to an instance."

Reply:
```suggestion
                description: Tags is an optional set of tag IDs to add to an instance.
```